pkg/collector/collector.go

```diff
@@ -17,14 +18,14 @@ import (
 )
 
 const (
-	apiPath      = "apis/eraser.sh/v1alpha1"
 	excludedPath = "/run/eraser.sh/excluded/excluded"
 )
 
 var (
 	runtimePtr    = flag.String("runtime", "containerd", "container runtime")
 	enableProfile = flag.Bool("enable-pprof", false, "enable pprof profiling")
 	profilePort   = flag.Int("pprof-port", 6060, "port for pprof profiling. defaulted to 6060 if unspecified")
+	scanDisabled  = flag.Bool("scan-disabled", true, "boolean for if scanner container is disabled")
```

Comment:
why are we disabling by default?

Reply:
the flag is always passed in from the controller,  but I'll change that since false would be more appropriate default